lib/codegen/ApplyPasses.cpp

```diff
@@ -145,4 +149,18 @@ void generate_object_file(llvm::Module &mod, llvm::raw_ostream &os) {
   pm.run(mod);
 }
 
+void do_bitcode_linking(llvm::Module &mod) {
```

Comment:
Same thing thing here. A small in-line comment explaining why we're doing that will be much appreciated!